Packs/CortexXDR/Playbooks/playbook-Cloud_Cryptomining.yml

```diff
@@ -0,0 +1,935 @@
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "32"
+    scriptarguments:
+      severity:
+        simple: High
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1310,
+          "y": 1210
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "28":
+    id: "28"
+    taskid: a1b6c2b4-a0b5-4d86-826e-ccb60dd05cbf
+    type: regular
+    task:
+      id: a1b6c2b4-a0b5-4d86-826e-ccb60dd05cbf
+      version: -1
+      name: Set the incident severity
+      description: Optionally increases the incident severity to the new value if it is greater than the existing severity.
+      scriptName: IncreaseIncidentSeverity
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "52"
+    scriptarguments:
+      severity:
+        simple: Low
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 170,
+          "y": 1370
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "29":
+    id: "29"
+    taskid: 33902d51-7869-4df3-8a19-41cf4fce8e97
+    type: condition
+    task:
+      id: 33902d51-7869-4df3-8a19-41cf4fce8e97
+      version: -1
+      name: Manual verdict verification
+      description: Manual verdict decision.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      APPROVED:
+      - "28"
+      UNAPPROVED:
+      - "27"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 720,
+          "y": 1040
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "32":
+    id: "32"
+    taskid: 758a229a-adc4-45bb-8a50-e3e115d05a09
+    type: regular
+    task:
+      id: 758a229a-adc4-45bb-8a50-e3e115d05a09
+      version: -1
+      name: Set SOC message for malicious activity
+      description: Sends an email using EWS.
+      script: '|||send-mail'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "36"
+    scriptarguments:
+      body:
+        simple: |-
+          Cortex XSOAR Notification
+
+          Cortex XSOAR has detected activity related to cryptomining.
+          After careful analysis, Cortex XSOAR found the activity malicious.
+
+          Please enter Cortex XSOAR incident ID ${incident.id} for further investigation and response.
+      subject:
+        simple: Cryptominig activity detected by Cortex XSOAR
+      to:
+        complex:
+          root: inputs.SOCEmailAddress
```

Comment:
what happens if the input is empty? lets check that first

Reply:
added comment in the playbook's input